tests/targets/test_android.py

```diff
@@ -41,10 +41,20 @@ def patch_os_isfile():
     return mock.patch("os.path.isfile")
 
 
+def patch_os_listdir():
+    return mock.patch("os.listdir")
+
+
```

Comment:
Nice, 4.9% more of coverage :smile:

**Note:** I think that we never use this mock, right?...are you planing to use it later?

Reply:
Yes good catch thanks. I actually started using it and decided no to for now.
I've removed it now